### DISCUSSION TITLE
US account balance

### DISCUSSION THREAD

**Discussion by alexandre1202** [DyegoCosta/trabalhando-remoto#182] [General]
olá Dyego.

Tudo bom?

Antes de mais nada eu realmente fico muito contente em ver outras pessoas que também gostam de compartilhar informações que não são muito fáceis de conseguir sem exigir algo em troca. Raridade! Parabéns pela iniciativa.

Contexto da minha pergunta:
Também comecei a trabalhar para uma empresa americana remotamente daqui de SP.
Eu receberei meu salário para minha conta do Itaú através desse Pilot.co, Pionner, PayPal etc..
Tenho planos de imigração para os US com um F1 daqui dois anos então um dia terei que levar o dinheiro e vou pagar IOF para isso, então pensei. Porque não usar a minha reserva em Real e deixar o salário em um banco dos US?
Ou seja, pensei em ir até os US, abrir uma conta corrente e pedir para a empresa me pagar nessa conta. Porém estou me perguntando como fica a questão do IR dos US. 

Não encontrei a resposta para essa pergunta quando tentei encontrar aqui no [Consulado](https://br.usembassy.gov/pt/u-s-citizen-services-pt/departamento-da-receita-federal-impostos-dos-eua/) 

Agradeço muito se por acaso você sabe me informar como funciona ou o contato de algum contador dos US que eu possa perguntar.

**Comment by lima-emanuel** [comment]
O que eu sei é que não precisa ir fisicamente aos EUA para abrir conta em banco. Até porque brasileiros não podem entrar lá por enquanto. E todo ano você teria que declarar imposto de renda lá e aqui e possivelmente pagar imposto duas vezes sobre a mesma renda. Então talvez compense deixar em reais e depois enviar para lá ou então abrir aquelas contas em dólar da BS2.

**Comment by alexandre1202** [comment]
Obrigado pela colaboração @emanuellima1 
Ajudou bastante. De fato você está correto. No momento as fronteiras estão bem fechadas para os brasileiros mas pensei em ir bem mais pra frente quando tudo se normalizar. 

Também já ouvi falar sobre essa bi-tributação e não quero dar viagem perdida então assim que eu conseguir uma fonte que formalize isso perante a receita federal dos US eu compartilho aqui.

Obrigado pela dica do BS2 (Banco de BHZ-MG). Mas quero um banco que não tenha registro do Banco Central. Tanto o BS2 quando o C6 oferecem essa proposta de ter valor em dólar na sua conta mas ambos cobram IOF, que é um imposto federal. O BS2 tem custo de 2% de Spread por transação.

A informação que eu tive da Pilot.co (concorrente a PayPal) é que não existe tributação para os US até USD 60k/y mas esse quero me certificar diretamente com os órgãos dos dois governos.

Mais uma vez obrigado pela colaboração

**Comment by samiraguiar** [comment]
@Alexandre1202 

Tem a [BB Americas](https://www.bbamericas.com/br/abra-sua-conta/), que é a filial do BB nos EUA. Abre pela internet mesmo, e se você tiver conta no BB o depósito inicial pedido é de US$ 3.000,00. Eles devem conseguir te orientar com a questão do IR.

Já ouvi falar também da [NomadGlobal](https://www.nomadglobal.com/), mas não conheço ninguém que usa.

Por fim, até onde eu sei, você só paga IR [no país em que reside](https://nomadcapitalist.com/2018/11/30/open-us-bank-account-non-resident/):
> Even non-residents’ US bank accounts are insured by the FDIC. In fact, the US generally treats foreigners the same as citizens in the realm of banking – except when it comes to taxes. If you’re not a US person for tax reasons, then you don’t need to pay tax on any interest you earn.

O Brasil tem acordo com os EUA, então muito dificilmente você pagaria imposto nos dois países.

**Comment by alexandre1202** [comment]
@samiraguiar 

Obrigado pelas novidades. Inclusive me fez lembrar de um amigo que já usa NomadGlobal.

Mais uma vez muito obrigado pela colaboração

**Comment by alexandre1202** [comment]
Olá @samiraguiar 

Estou passando para atualizar que realmente o Nomad é uma opção de banco americano que permite a pessoa ter uma conta corrente em USD mas essa conta é vinculada ao Brasil.
![image](https://user-images.githubusercontent.com/8005205/115075503-8dd78f80-9ed1-11eb-8c26-bcbafc14ecab.png)


Como ainda não tenho informação de um especialista então preciso continuar buscando outras fontes. Veja:
![image](https://user-images.githubusercontent.com/8005205/115075769-f888cb00-9ed1-11eb-8f06-235560cd5673.png)

**Comment by samiraguiar** [comment]
@Alexandre1202 obrigado pela atualização.

Então você quer uma conta que não seja ligada aos órgãos reguladores brasileiros?
Achei que apenas uma conta nos EUA já lhe servisse.

Em todo caso, do ponto de vista legal, enquanto você for residente terá que declarar a conta no exterior no seu imposto de renda, além de saldo e em algumas situações ganhos de capital. Mesmo que opte por omitir isso, não é difícil o rastreio enquanto você estiver mandado para cá o dinheiro dessa conta. E em casos graves, como processos judiciais, o governo pode solicitar o bloqueio também de contas _off-shore_ em países com os quais tem acordo. Não é um processo trivial e que seria feito em qualquer situação, mas é possível. Pelo que entendi, porém, você não pretende trazer o dinheiro pra cá, mas deixar lá e viver aqui com o que tem guardado em Reais. Não me ficou claro se você tem receio de bloqueios injustificáveis de contas (à la Collor) ou se não deseja prestar contas à Receita, mas fica a ressalva de que não declarar é sonegação fiscal.

Tem algum problema com a Payoneer? Ela te dá uma conta nos EUA, que até onde eu sei tem proteção do FIDC, e você pode solicitar um cartão pré-pago que funciona para saques e pagamentos lá nos EUA. Pagando e sacando diretamente em dólares deve ter uma taxa bem pequena. Além disso, eles não estão sujeitos aos órgãos de controle brasileiros.

A maior limitação da Payoneer que vejo em relação a uma conta comum é sobre enviar e receber dinheiro. Somente dá pra receber dinheiro de contas empresariais, e pra enviar para contas de terceiros em alguns casos eles usam intermediários. Eu **acho** que quando você transfere para uma conta na mesma moeda, como transferir da sua conta Payoneer USD para uma conta nos EUA, eles não usam intermediários, mas teria que testar ou perguntar ao suporte. Se isso estiver correto, você poderia enviar uma parte do salário para a [TD Ameritrade](https://www.tdameritrade.com/home.page) e deixar investido em ativos rentáveis como ETFs e outros, evitando perder patrimônio. Essa corretora é 100% americana sem vínculos com o Brasil (diferentemente da Avenue ou da Stake, por exemplo), e eles permitem a não-residentes abrirem conta remotamente via correios ou FAX (Internet não).

**Comment by alexandre1202** [comment]
@samiraguiar 

Cara! Estou muito surpreso com essa riqueza de detalhes. A paciência e o tempo gasto para me escrever tem um valor muito grande pra mim. Antes de mais nada, muito obrigado mesmo! 

Obrigado por avisar sobre essas questões do fisco. Eu quero sempre estar dentro da lei porque não quero ter problemas.
Acredito que uma explicação em uma conversa eu explicaria melhor. 

**O que eu realmente quero é simplesmente usar minhas economias em R$ das quais já paguei IRRF e deixar o dinheiro no banco americano desde que eu não tenha bi-tributação, caso um dia eu transfira uma parte para o BR.** 

Agora com todos esses detalhes, a minha pergunta é:
- Em 2022 como terei que lançar o meu redimento no IR?

Sem um Informe de Rendimento eu não sei como declarar então por isso falei que eu não queria nenhum vínculo com o governo e autoridades brasileiras. Além disso eu realmente tenho medo do que já vi acontecendo com minha família e amigos nos tempos Colloridos... 
Mas agora ficou claro que enquanto eu não me declarar que já sai do Brasil, ou seja, enquanto eu viver aqui eu devo declarar no IR. Mas ainda não está claro se além do carne leão que terei que pagar no caso de transferências US$ -> BR$ como fica o IR sobre o meu salário que ficará no banco americano, dentro da lei.

Enquanto eu não tenho uma conta que eu tenha aberto diretamente lá nos US eu penso em inicialmente abrir uma conta virtual em um banco americano, receber o meu salário já direto nessa conta mensalmente e se possível aplicar para meus planos futuros. Mas não faço ideia de como declarar isso no BR até que eu declare minha saida do BR.

Por favor, consegue me ajudar?

**Comment by samiraguiar** [comment]
@Alexandre1202 

Se você vai ficar como residente no Brasil, não precisa pagar imposto duas vezes. Eu nunca prestei serviço para empresas americanas, mas até onde eu sei, você vai precisar preencher um formulário chamado [W-8BEN](https://cartaoprepago.net/como-preencher-formulario-w8ben/). Esse formulário serve para dizer para o fisco americano que você é um estrangeiro não-residente e que não vai pagar os impostos pra eles, mas sim no seu país de origem. Você entrega esse formulário preenchido para a empresa para a qual você presta serviço, que vai retê-lo caso o fisco solicite. O pessoal discute sobre esse formulário [aqui](https://github.com/DyegoCosta/trabalhando-remoto/issues/68).

Os valores que você recebe vão ser declarados no seu carnê-leão normalmente, e aí você paga o imposto aqui independente de trazer o dinheiro para cá. Já na declaração anual do IRPF, a sua conta no exterior, bem como o saldo, [terão](https://www.remessaonline.com.br/blog/como-declarar-minha-conta-no-exterior-no-imposto-de-renda-2020/) que ser informados na ficha de "bens e direitos", categoria "depósito bancário em conta corrente no exterior". Se o dólar se valorizou, você declara esses rendimentos na ficha "Rendimentos Isentos e Não Tributáveis", com código "26 - Outros". Vale notar que se os valores na conta do exterior passarem de US$ 100 mil, tem que fazer a DCBE.

Quanto aos valores que você manda pra cá da sua conta do exterior, não acho que você deva imposto de renda novamente porque você já pagou pelo carnê-leão. Pode ter que pagar o IOF a depender do serviço que usar para a transferência, mas aí é porque o fato gerador desse imposto é a operação de transferência em si. Já se você investir nos EUA em ações ou ETFs, por exemplo, e receber dividendos, vai pagar 30% de impostos que são retidos na fonte, e aí não paga aqui. Quando vender os ativos de mesmo tipo e tiver lucro, paga 15% de imposto se o volume de venda passar de R$ 35 mil.

Agora se você fizer a declaração de saída definitiva do Brasil, se mudar para os EUA, receber dólares e pagar imposto lá, e depois voltar para o Brasil e trazer esse dinheiro para cá... aí já não faço ideia. :smile:

**Comment by alexandre1202** [comment]
oi @samiraguiar 
Muito obrigado mais uma vez.
Ontem eu falei com o diretor da empresa que me contratou e aproveitei pra perguntar sobre tax income e ele também confirmou que para essa situação específica eu não pago imposto lá. Disse que pelo fato de se tratar de um trabalho remoto, eu não ser americano e fisicamente fora dos US eu não tenho que pagar por se tratar de um "contractor" ou seja, um terceiro. 

O único ponto que ainda não ficou claro pra mim é se realmente eu devo pagar o IR / carne leão de um valor que eu se quer transferi para o Brasil. Ou seja, como vou pagar em reais se eu nem converti o dinheiro, entende?
Mas uma coisa por vez...rs... muito obrigado.

**Comment by samiraguiar** [comment]
@Alexandre1202 

> O único ponto que ainda não ficou claro pra mim é se realmente eu devo pagar o IR / carne leão de um valor que eu se quer transferi para o Brasil.

Eu entendo que sim. No [documento](https://www.gov.br/receitafederal/pt-br/acesso-a-informacao/perguntas-frequentes/declaracoes/dirpf/pr-irpf-2021-v-1-1-2021-03-05.pdf) de perguntas e respostas do IRPF 2021 tem essa questão:

> RESIDENTE NO BRASIL -RENDIMENTOS DO EXTERIOR
> 118 - Como são tributados os rendimentos recebidos por residente no Brasil de fontes no exterior?
> ...
> 3 - Demais rendimentos recebidos
> Os demais rendimentos recebidos de fontes situadas no exterior por residente no Brasil, **transferidos ou não para o País**, estão sujeitos à tributação sob a forma de recolhimento mensal obrigatório (carnê-leão), no mês do recebimento, e na Declaração de Ajuste Anual.

(vale dar uma olhada nas outras perguntas desse documento, em especial a 123 que fala sobre transferência de residência para o Brasil)

E dá uma olhada também nas issues #85, #35 e #58 que falam sobre cenários parecidos com esse seu.

**Comment by alexandre1202** [comment]
Olá @samiraguiar 
Tudo bom?

Depois de ler os assuntos que apontou e outros também importantes para estar regular eu estou em busca de informações ou contratação de um [CPA](https://www.taxesforexpats.com/?gclid=Cj0KCQjwppSEBhCGARIsANIs4p6sPdTJzCAOZPATOQs7LamEGB814t7fDM4ycw1_uq8qHzn-QdJJpy8aAovlEALw_wcB) para me informar sobre a abertura de uma empresa nos EUA.

Agradeço de alguém tiver uma indicação De confiança

**Comment by MARIOGOMES** [comment]
@samiraguiar e todos

Bom dia. 
Excelente os esclarecimentos.

Um questionamento que nao observei referencia :
 " Já na declaração anual do IRPF, a sua conta no exterior, bem como o saldo, terão que ser informados na ficha de "bens e direitos", categoria "depósito bancário em conta corrente no exterior". "

No caso de recebimento pela prestação de serviços no exterior em conta Payonner, devemos pagar o Carne-Leao conforme citado anteriormente e no caso de parte dos recursos serem transferidos para o Brasil e parte permanecerem em conta Payonner : Seria possível declarar esse saldo em US$, em uma conta Payonner, em 31/12 ? A RFB aceita uma conta na Payonner ? Como seria a descrição ? 

Agradeço antecipadamente pelo esclarecimentos.

**Comment by alexandre1202** [comment]
Bom dia a todos


A informação que eu tenho do contador eh que no caso de brasileiros e
residentes aqui e que são “contractors” ou seja terceiros e que estão
fechando contrato com firma para essa finalidade então basta recolher o
DARF mensalmente do IRPJ e o outro imposto que não lembro agora mas que
somam 4% e estamos regular com a RF do BR e dia US.

Mensalmente eu também terei que enviar a “invoice” para a empresa que me
contratou com uma numeração sequencial que por sua vez deve ser
referenciada sempre que for recolher o DARF da IRPJ.

Quando chegar a declaração anual esses recolhimentos serão referenciados
nessa DIRPJ anual sim.

Fico a disposição mas o @samiraguiar pode fazer um double check

Espero ter ajudado

Alexandre

On Fri, 30 Apr 2021 at 09:43 Gomid ***@***.***> wrote:

> @samiraguiar <https://github.com/samiraguiar> e todos
>
> Bom dia.
> Excelente os esclarecimentos.
>
> Um questionamento que nao observei referencia :
> " Já na declaração anual do IRPF, a sua conta no exterior, bem como o
> saldo, terão que ser informados na ficha de "bens e direitos", categoria
> "depósito bancário em conta corrente no exterior". "
>
> No caso de recebimento pela prestação de serviços no exterior em conta
> Payonner, devemos pagar o Carne-Leao conforme citado anteriormente e no
> caso de parte dos recursos serem transferidos para o Brasil e parte
> permanecerem em conta Payonner : Seria possível declarar esse saldo em US$,
> em uma conta Payonner, em 31/12 ? A RFB aceita uma conta na Payonner ? Como
> seria a descrição ?
>
> Agradeço antecipadamente pelo esclarecimentos.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DyegoCosta/trabalhando-remoto/issues/133#issuecomment-830068029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB5CMVIYGQPJQFMBKMJGKCTTLKQWHANCNFSM43AK7NSA>
> .
>
-- 
------------------------- Alexandre Barbosa 11-99169-3870 (Claro)

**Comment by samiraguiar** [comment]
@MARIOGOMES

A descrição eu creio que possa ser os dados da conta na moeda que você deixou acumular, e também o "Customer ID" da Payoneer. Outra possibilidade talvez seria colocar na categoria "99 - Outros bens e direitos" e explicar que é uma carteira virtual, sem país e número de conta. É fato que a Payoneer não presta contas ao governo brasileiro e essas contas que ela fornece são pra recebimentos via intermediários, então provavelmente se o governo quisesse verificar seria um pouco mais trabalhoso do que com bancos comuns. Mas no meu entender a questão é mais manter a receita informada de que você possui aquele valor pra não causar estranhezas, e se houver qualquer dúvida eles sempre entram em contato solicitando mais comprovação. Então mais importante que a descrição ou categoria, é colocar os valores corretos.

Por exemplo, vamos supor que em Fevereiro de 2019 o cara comprou R$ 5 mil em dogecoin, deixou numa hardware wallet em casa e não informou na DIRPF de 2020 (quando criptos nem tinham uma categoria específica). O valor é baixo demais pra chamar atenção, e a wallet obviamente não presta informações à receita, então ninguém sabe que ele tem essas criptos. Daí chegou Abril de 2021 e o cara percebeu que tinha agora R$ 1,7 milhões. Como vai explicar que tinha essas coins antes e foi tudo variação cambial? Vai dar rolo, multa, etc. Claro que esse caso é mais extrapolado, mas acho que a ideia é a mesma: manter a receita informada de valores, bens, ganhos, etc.

**Comment by MARIOGOMES** [comment]
@Alexandre1202 e @samiraguiar 

Obrigado pelos esclarecimentos.

@Alexandre1202, normalmente utilizo para calculo do IRPF para o Caret-Leao, o link da RFB, http://www26.receita.fazenda.gov.br/irpfsimulaliq/private/pages/simuladoraliquota.jsf e a aliquota no meu caso, nao eh a de 4%, como referenciado por voce anteriormente.

@samiraguiar, concordo com sua linha de raciocínio em informar os valores corretos na Declaração de Ajuste Anual, pois alem das variacoes pelo seu exemplo com cripto, podemos ter desvalorização do Real, aumentando o valor do ativo no exterior em Reais.

Muito obrigado a todos mais uma vez
Abs

> @MARIOGOMES
> 
> A descrição eu creio que possa ser os dados da conta na moeda que você deixou acumular, e também o "Customer ID" da Payoneer. Outra possibilidade talvez seria colocar na categoria "99 - Outros bens e direitos" e explicar que é uma carteira virtual, sem país e número de conta. É fato que a Payoneer não presta contas ao governo brasileiro e essas contas que ela fornece são pra recebimentos via intermediários, então provavelmente se o governo quisesse verificar seria um pouco mais trabalhoso do que com bancos comuns. Mas no meu entender a questão é mais manter a receita informada de que você possui aquele valor pra não causar estranhezas, e se houver qualquer dúvida eles sempre entram em contato solicitando mais comprovação. Então mais importante que a descrição ou categoria, é colocar os valores corretos.
> 
> Por exemplo, vamos supor que em Fevereiro de 2019 o cara comprou R$ 5 mil em dogecoin, deixou numa hardware wallet em casa e não informou na DIRPF de 2020 (quando criptos nem tinham uma categoria específica). O valor é baixo demais pra chamar atenção, e a wallet obviamente não presta informações à receita, então ninguém sabe que ele tem essas criptos. Daí chegou Abril de 2021 e o cara percebeu que tinha agora R$ 1,7 milhões. Como vai explicar que tinha essas coins antes e foi tudo variação cambial? Vai dar rolo, multa, etc. Claro que esse caso é mais extrapolado, mas acho que a ideia é a mesma: manter a receita informada de valores, bens, ganhos, etc.

**Comment by MARIOGOMES** [comment]
Boa tarde
@Alexandre1202 e @samiraguiar

Supondo um faturamento de serviços prestados ao exterior, em um determinado mes de 2020, cuja a conversao para reais utilizando o valor do dolar do dia 15 do mes anterior, nos da um valor de R$ 20.000,00. Levando em consideração o pagamento do INSS, a época, pelo teto cujo o valor resulta em R$ 1220,21, qual  seria a base de calculo para emissão do DARF ?

Agradeco possiveis esclarecimentos.
Abs

**Comment by alexandre1202** [comment]
olá Mario.
A sua dúvida é bastante específica e não faço ideia de onde deve ter essa informação. Com certeza nas atualizações da IOB ou coisa parecida que os contadores usam deve ter. Acredito que um bom contador de sua confiança com conhecimento das obrigações da RF Brasil pode te ajudar com isso. Ou o @samiraguiar que também conhece bem mais do que eu.

**Comment by samiraguiar** [comment]
@MARIOGOMES se não me engano esse valor do INSS é abatido da base de cálculo porque você pegou o teto, mas eu não entendo muito de carnê-leão porque faço como PJ. Dá uma olhada nas issues https://github.com/DyegoCosta/trabalhando-remoto/issues/69 e https://github.com/DyegoCosta/trabalhando-remoto/issues/58, acho que elas te ajudam.

**Comment by MARIOGOMES** [comment]
Obrigado @Alexandre1202, vou consultar um contador sim.
@samiraguiar, olhando as issues #69 e # 58 os mesmos sao esclarecedores quanto ao meu questionamento anterior.

Obrigado a todos
